server/src/main/java/org/opensearch/node/NodeService.java

```diff
@@ -96,6 +97,7 @@ public class NodeService implements Closeable {
     private final FileCache fileCache;
     private final TaskCancellationMonitoringService taskCancellationMonitoringService;
     private final RepositoriesService repositoriesService;
+    AdmissionControlService admissionControlService;
```

Comment:
can we make this `private final` as well ?

Reply:
Updated.